test/form-component-spec.js

```diff
@@ -3,1609 +3,1710 @@ import { assert } from 'chai';
 import React from 'react';
```

Comment:
The majority of the changes here are just adding the testContexts though there were some little refactor/cleanup changes which are easier to find by looking at the commits than trying to find in this diff.